compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/SkiaBasedOwner.skiko.kt

```diff
@@ -392,6 +396,20 @@ internal class SkiaBasedOwner(
         }
     }
 
+    /**
+     * If pointerPosition is UIKitInterop, then Compose skip touches. And touches goes to UIKit view
+     */
+    fun hitInteropView(pointerPosition: Offset, isTouchEvent: Boolean): Boolean {
+        val result = HitTestResult<LayoutModifierNode>()
+        pointerInputEventProcessor.root.hitTestLayout(pointerPosition, result, isTouchEvent)
+        val last = result.lastOrNull()
+        return if (last is BackwardsCompatNode) {
+            last.element is InteropViewCatchPointerModifier
+        } else {
+            false
+        }
```

Comment:
If I understand correctly, this function will always return `false` for desktop and web, right?
At least for now, because they don't use `InteropViewCatchPointerModifier ` ?
Maybe it makes sense to make expect/actual implementations of this function. This would help to avoid redundant calls on other platforms.

Reply:
Yes, this function will always return false for Desktop and Web.
For now, this function calls only on UIKit target, because it calls on:
```Kotlin
            pointInside = { point, _ ->
                !layer.hitInteropView(point, isTouchEvent = true)
            },
```
So there are no redundant calls before Desktop and Web not call it.